timemachine/fe/atom_mapping.py

```diff
@@ -64,9 +115,18 @@ def mcs_map_graph_only_complete_rings(a, b, timeout: int = 3600, smarts: Optiona
         matchChiralTag=False,
         atomCompare=Chem.rdFMCS.AtomCompare.CompareAny,
         bondCompare=Chem.rdFMCS.BondCompare.CompareAny,
+        seedSmarts=smarts if type(smarts) == str else "",
```

Comment:
nit: `smarts or ""`

Reply:
Applied in https://github.com/proteneer/timemachine/pull/812/commits/c8f253662ec35f6c44a7b0718c761c88e8622479